apps/docs/src/app/platform/component-docs/platform-split-menu-button/platform-split-menu-button-examples/platform-split-menu-button-examples.component.html

```diff
@@ -68,7 +69,20 @@ <h2>Compact Split Menu Button</h2>
 <span>&nbsp;&nbsp;Disabled</span>
 <br />
 <br />
-<div>Selected Item: {{ selectedCompactItem }}</div>
+<p>Selected Item: {{ selectedCompactItem }}</p>
+
+<h3>Internationalisation support with i18n directive</h3>
+<fdp-split-menu-button
+    [id]="'split-menu-button4'"
+    [name]="'split-menu-button4'"
+    [menu]="basicMenuCompact"
+    [buttonLabel]="label"
+    [icon]="'world'"
+    [title]="'default button with menu'"
+    (primaryButtonClick)="onPrimaryButtonClickCompact()"
+    i18n
```

Comment:
if giving example for i18n, please give an id to the i18n marker at the least. eg:`i18n="@@basicSplitMenuBtn"`. 
You can also show how `title` and `buttonLabel` can be marked with i18n prefixed markers.

Reply:
added in example.